tests/test_gather_financial_statement_time_series.py

```diff
@@ -1,94 +1,141 @@
 # Author: Team Andrey Markov
 # tests for gather_financial_statement_time_series
 
-## Reference: https://ubc-mds.github.io/py-pkgs/testing.html#pytest-tests
+# Reference: https://ubc-mds.github.io/py-pkgs/testing.html#pytest-tests
 
 from pyntrinio.pyntrinio import gather_financial_statement_time_series
 from pytest import raises
+import pandas as pd
 
 # Sample data for testing
 api_key = 'OjQ0YzljN2E4ODk5YzM1MzVhMTZmNTUwNmE2N2M0NTYz'
 ticker = 'CVX'
 statement = 'income_statement'
 year = ['2017', '2018']
-period = ['Q1','Q3']
+period = ['Q1', 'Q3']
+var_1 = OjQ0Yz
+var_2 = CVX
+var_3 = years
+var_4 = income_statement
+var_5 = Q1
```

Comment:
Why did you create those variables? why didn't you just put them in the code?

Reply:
- As discussed, addressed this issue in the latest commit